tools/standalone-binary-distro/make_distro_builds.sh

```diff
@@ -34,6 +36,9 @@ carthage bootstrap --no-build
 
 echo "carthage build..."
 carthage build --platform iOS --use-xcframeworks --no-skip-current
+# reset trap
+trap - INT TERM HUP EXIT
+rm -f "$xcconfig"; mv .dependency-manager-tests/ dependency-manager-tests/ ;
```

Comment:
Why resetting the trap? It makes it unclear on how to maintain this script for next changes - should we let the trap undo state changes, or do it by hand after all? What if the script fails after trap is reset, but not all changes are reset?